linera-core/src/client/mod.rs

```diff
@@ -1822,7 +1822,7 @@ where
                 chain_state_view
                     .manager
                     .get()
-                    .pending_blobs
+                    .proposed_pending_blobs
```

Comment:
Not introduced in this PR but is it necessary to request `self.chain_state_view()` on every iteration of `blob_ids` loop?